.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+contact_links:
```

Comment:
If we don't want to allow free-form issues, we can add `blank_issues_enabled: false` at the beginning of the file.

Reply:
Yeah, as you say, maybe it's best to leave them enabled. From experience with triaging at mypy, most new contributors don't spot the small "Don’t see your issue here? [Open a blank issue.](https://github.com/python/mypy/issues/new)" link at the bottom, so I don't think we need to explicitly disable opening blank issues, and core devs will probably appreciate the option of opening a blank issue.